modules/ossm-installing-multi-primary-multi-network-topology.adoc

```diff
@@ -0,0 +1,294 @@
+----
+
+.. Deploy the `helloworld` application:
+
+... Create the `helloworld` service by running the following command:
++
+[source,terminal]
+----
+oc apply --context="${CTX_CLUSTER1}" \
+  -f https://raw.githubusercontent.com/istio/istio/${ISTIO_VERSION}/samples/helloworld/helloworld.yaml \
+  -l service=helloworld -n sample
+----
+
+... Create the `helloworld-v1` deployment by running the following command:
++
+[source,terminal]
+----
+oc apply --context="${CTX_CLUSTER1}" \
+  -f https://raw.githubusercontent.com/istio/istio/${ISTIO_VERSION}/samples/helloworld/helloworld.yaml \
+  -l version=v1 -n sample
+----
+
+.. Deploy the `sleep` application by running the following command:
++
+[source,terminal]
+----
+oc apply --context="${CTX_CLUSTER1}" \
+  -f https://raw.githubusercontent.com/istio/istio/${ISTIO_VERSION}/samples/sleep/sleep.yaml -n sample
+----
+
+. Wait for the sample applications on the East cluster to return the "Ready" status condition by running the following command:
++
+[source,terminal]
+----
+$ oc --context="${CTX_CLUSTER1}" wait --for condition=available -n sample deployment/helloworld-v1
+$ oc --context="${CTX_CLUSTER1}" wait --for condition=available -n sample deployment/sleep
+----
+
+. Deploy the sample applications on the West cluster:
+
+.. Create a sample application namespace on the West cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get ns sample --context "${CTX_CLUSTER2}" || oc create --context="${CTX_CLUSTER2}" namespace sample
+----
+
+.. Label the application namespace to support sidecar injection by running the following command:
++
+[source,terminal]
+----
+$ oc label --context="${CTX_CLUSTER2}" namespace sample istio-injection=enabled
+----
+
+.. Deploy the `helloworld` application:
+
+... Create the `helloworld` service by running the following command:
++
+[source,terminal]
+----
+oc apply --context="${CTX_CLUSTER2}" \
+  -f https://raw.githubusercontent.com/istio/istio/${ISTIO_VERSION}/samples/helloworld/helloworld.yaml \
+  -l service=helloworld -n sample
+----
+
+... Create the `helloworld-v2` deployment by running the following command:
++
+[source,terminal]
+----
+oc apply --context="${CTX_CLUSTER2}" \
+  -f https://raw.githubusercontent.com/istio/istio/${ISTIO_VERSION}/samples/helloworld/helloworld.yaml \
+  -l version=v2 -n sample
+----
+
+.. Deploy the `sleep` application by running the following command:
++
+[source,terminal]
+----
+oc apply --context="${CTX_CLUSTER2}" \
+  -f https://raw.githubusercontent.com/istio/istio/${ISTIO_VERSION}/samples/sleep/sleep.yaml -n sample
+----
+
+. Wait for the sample applications on the West cluster to return the "Ready" status condition by running the following command:
++
+[source,terminal]
+----
+$ oc --context="${CTX_CLUSTER2}" wait --for condition=available -n sample deployment/helloworld-v2
+$ oc --context="${CTX_CLUSTER2}" wait --for condition=available -n sample deployment/sleep
```

Comment:
This can be converted into a single command (so you don't have to split this snippet into two):
```suggestion
$ oc --context="${CTX_CLUSTER2}" wait --for condition=available -n sample \
     deploy/helloworld-v2 deploy/sleep
```